hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -424,17 +433,17 @@ public AbfsRestOperation acquireLease(final String path, int duration, TracingCo
 
     final URL url = createRequestUrl(path, abfsUriQueryBuilder.toString());
     final AbfsRestOperation op = new AbfsRestOperation(
-        AbfsRestOperationType.LeasePath,
-        this,
-        HTTP_METHOD_POST,
-        url,
-        requestHeaders);
+            AbfsRestOperationType.LeasePath,
+            this,
+            HTTP_METHOD_POST,
+            url,
+            requestHeaders);
```

Comment:
Many space changes, will need to be revoked.

Reply:
Unintended edits, revoking in latest commit.